pages/rops/update/views/components/confirm.jsx

```diff
@@ -107,7 +107,7 @@ export default function Confirm() {
           <dd>{yn(rop.proceduresCompleted)}</dd>
 
           <dt>Only protected embryonic forms used</dt>
-          <dd>{yn(rop.postnatal)}</dd>
+          <dd>{yn(rop.postnatal === null ? null : !rop.postnatal)}</dd>
```

Comment:
How come this particular field gets this treatment and not all the other `yn` fields that might be null?

Reply:
`yn` function takes null into account, however the boolean is switched for this field (for reasons) so `true` needs to show `'Yes'`, `false` `'No'` and `null` `'-'`

Reply:
we could do something like `typeof rop.postnatal === 'boolean' ? !rop.postnatal : rop.postnatal` if you think it's clearer

Reply:
```suggestion
          // need to reverse the boolean for postnatal animals because the phrasing of the question changes
          <dd>{yn(rop.postnatal === null ? null : !rop.postnatal)}</dd>
```